### PR TITLE
Add gallery example for OEDI system 9068

### PR DESCRIPTION
**Pull request by kandersolar**
- A step towards #1952
 - [x] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - ~[ ] Tests added~
 - ~[ ] Updates entries in [`docs/sphinx/source/reference`](https://github.com/pvlib/pvlib-python/blob/main/docs/sphinx/source/reference) for API changes.~
 - [x] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/main/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - ~[ ] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.~
 - [x] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels (including `remote-data`) and Milestone are assigned to the Pull Request and linked Issue.

### CODE REVIEW

**Review by AdamRJensen — APPROVED** [review]
I would expose the other loss parameters:

        # Other loss types are left at their default amounts.
        losses_parameters = dict(shading=0, availability=0)

Otherwise things look good to me.
image

I have my doubts if applying a transient operation on the module temperature makes this any more accurate when you use satellite data?

A shame that this great data initiative has so poor metadata - a bit ironic really.

File: docs/examples/system-models/plot_oedi_9068.py [hunk 1]
```diff
@@ -0,0 +1,282 @@
+"""
+SR_CO (OEDI System 9068)
+========================
+
+A basic model of a 4.7 MW single-axis tracking CdTe system located in
+Colorado, United States.
```
Comment (AdamRJensen) [inline docs/examples/system-models/plot_oedi_9068.py:6]:
```suggestion
Modeling of a basic model of a 4.7 MW single-axis tracking CdTe system
located in Colorado, United States.
```

File: docs/examples/system-models/plot_oedi_9068.py [hunk 2]
```diff
@@ -0,0 +1,282 @@
+"""
+SR_CO (OEDI System 9068)
```
Comment (AdamRJensen) [inline docs/examples/system-models/plot_oedi_9068.py:2]: Can we make a more descriptive title? 4.7 MW CdTe single-axis tracking

**Comment by kandersolar** [comment]
> I have my doubts if applying a transient operation on the module temperature makes this any more accurate when you use satellite data?

5-minute PSM3 data is very "spiky", so using a transient model at least makes the modeled Tcell more realistic/smooth, if not also more accurate.  IMHO it's reasonable to include by default unless we know for some reason that it makes results worse.  I think SAM applies it by default for any dataset with time step below 20 minutes. 

> Maybe add a note about why you're not using the measured irradiance (I think that would be useful for people)

It does say this before fetching the weather dataset:
> The system does have measured plane-of-array irradiance data, but the measurements suffer from row-to-row shading and tracker stalls. In this example, we will use weather data taken from the NSRDB PSM3 for the year 2019.

**Comment by adriesse** [comment]
> The system does have measured plane-of-array irradiance data, but the measurements suffer from row-to-row shading and tracker stalls. In this example, we will use weather data taken from the NSRDB PSM3 for the year 2019.

It could actually be instructive to use that data in the example anyway.

**Pull request merged**